tools/capable.py

```diff
@@ -150,8 +176,11 @@ def print_event(cpu, data, size):
     print("%-9s %-6d %-6d %-16s %-4d %-20s %d" % (strftime("%H:%M:%S"),
         event.uid, event.pid, event.comm.decode('utf-8', 'replace'),
         event.cap, name, event.audit))
+    if args.kernel_stack:
+        print_stack(bpf, event.kernel_stack_id, -1)
```

Comment:
You always pass -1 with tgid for print_stack. Any issues here?

Reply:
I was going to add also the user-space stack, so print_stack() is already prepared to resolve both kernel stack and user stack. I'll add the option to print the user stack soon in the next change.

Reply:
Yes, we should print user stack as well in parity with kernel stack print. Then in print_stack
```
print_stack(bpf, stack_id, tgid):
```
You will need to change to
```
print_stack(bpf, kern_stack_id, user_stack_id, tgid):
```
In this case, tgid should be real tgid.